requirements-notebooks.txt

```diff
@@ -1,4 +1,4 @@
-torch==1.13.1
+torch==2.0.1
```

Comment:
Could you also add the minimum version to the `extras_require` field inside setup.py?